spacy_llm/backends/rest/registry.py

```diff
@@ -12,7 +12,8 @@ def backend_rest(
     config: Dict[Any, Any] = SimpleFrozenDict(),
     strict: bool = True,
     max_tries: int = 3,
-    timeout: int = 30,
+    interval: int = 30,
```

Comment:
interval should probably start smaller since we're doubling it each time, ideally start at like 1 second and make max_tries a little higher (5 maybe)